osg-koji

```diff
@@ -264,11 +270,18 @@ def setup_koji_client_cert(user_cert, user_key):
             not os.path.samefile(KOJI_USER_CONFIG_DIR, OSG_KOJI_USER_CONFIG_DIR)) and
             os.path.isfile(OLD_CLIENT_CERT_FILE)):
 
-        if ask_yn("Copy client cert from '" + OLD_CLIENT_CERT_FILE + "' ?"):
+        if ask_yn("""
+You already have a client cert at '%s'.
+Reuse that file?
+""" % OLD_CLIENT_CERT_FILE):
             copy_old_client_cert(new_client_cert_path)
             return
 
-    if ask_yn("Set up symlink to grid proxy?"):
+    if ask_yn("""
+Set up symlink to grid proxy? If you do this, you can use grid-proxy-init or
+voms-proxy-init to create a proxy and not have to type your password in as
+often.
+"""):
```

Comment:
[For lines above, not below…] Symlink to your expected grid proxy location? Doing so means that you can use a proxy (from grid- or voms-proxy-init) for authentication and thereby not need to type a password in as often.